src/main/java/rx/internal/util/unsafe/SpscArrayQueue.java

```diff
@@ -118,19 +118,22 @@ public SpscArrayQueue(final int capacity) {
      */
     @Override
     public boolean offer(final E e) {
```

Comment:
Are the fixes for SpscArrayQueue tied to this PR or can they be done separately?